aliskin/assignment1/anagram.py

```diff
@@ -0,0 +1,19 @@
+from collections import defaultdict
+
+def isAnagram(s, t, case_sensitive=True):
+    """
+    Given two strings s and t returns True if t is anagram of s and False otherwise.
+    """
+    if len(s) != len(t):
+        return False
+    if (not case_sensitive):
+        s = s.casefold()
+        t = t.casefold()
+    symbols = defaultdict(int)
+    for c in s:
```

Comment:
You can also use a collections.Counter object to count occurrences of characters in words: https://docs.python.org/2/library/collections.html#collections.Counter